app/views/users/_form.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= form_with(model: user, local: true) do |form| %>
+  <% if user.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(user.errors.count, "error") %> prohibited this user from being saved:</h2>
+
+      <ul>
+      <% user.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= form.hidden_field :organization_id, value: current_user.organization.id %>
```

Comment:
I think that with this https://github.com/rubyforgood/abalone/pull/381/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R37 you don't need the hidden field anymore